shared/src/store/action-reducers/simulation.ts

```diff
@@ -1285,12 +1783,40 @@ export namespace SimulationActionReducers {
                 draftState,
                 { simulatedRegionId, behaviorId, managedSimulatedRegionId }
             ) {
+                const simulatedRegion = getElement(
+                    draftState,
+                    'simulatedRegion',
+                    simulatedRegionId
```

Comment:
This tag is duplicated if a region is managing itself

Reply:
d